pages/entropy/generate-random-numbers/evm.mdx

```diff
@@ -14,81 +14,76 @@ npm install @pythnetwork/entropy-sdk-solidity
 
 ## Setup
 
-To use the SDK, you need the address of an Entropy contract on your blockchain.
+The SDK exports a `IEntropyConsumer` interface that your contract should implement. The interface
```

Comment:
You can also mention the IEntropy interface here and add it as an import in the next code snippet.